src/core/recv_buffer.c

```diff
@@ -164,14 +181,66 @@ QuicRecvBufferHasUnreadData(
 }
 
 _IRQL_requires_max_(DISPATCH_LEVEL)
-void
-QuicRecvBufferIncreaseVirtualBufferLength(
+BOOLEAN
+QuicRecvBufferTryIncreaseVirtualBufferLength(
```

Comment:
Missed one caller of the old name:
```
D:\a\msquic\msquic\src\core\stream_recv.c(805,17): warning C4013: 'QuicRecvBufferIncreaseVirtualBufferLength' undefined; assuming extern returning int [D:\a\msquic\msquic\build\windows\x86_openssl\src\core\core.vcxproj]
```

Reply:
I missed a file when pushing that iteration, when filtering out the changes for the API part. Fixed it.